invenio_damap/assets/semantic-ui/js/invenio_damap/index.js

```diff
@@ -8,18 +8,37 @@
 import ReactDOM from "react-dom";
 import React from "react";
 import { Grid } from "semantic-ui-react";
-import { DMPButton } from "./DMPButton";
 
-const element = document.getElementById("invenio-damap-render");
+import { DMPAddButton } from "./DMPAddButton";
+import { DMPAuthButton } from "./DMPAuthButton";
 
-const recordManagementAppDiv = document.getElementById("recordManagement");
-const record = JSON.parse(recordManagementAppDiv.dataset.record);
+const element = document.getElementById("dmps-container");
 
-// the render element won't be available if we're not the record owner
-element &&
+if (element) {
+  const availableDmps = element.dataset.dmps;
+  const damapUrl = element.dataset.damapUrl;
+
+  const parsedAvailableDmps =
+    availableDmps && availableDmps !== "[]" ? JSON.parse(availableDmps) : [];
```

Comment:
```suggestion
    availableDmps ? JSON.parse(availableDmps) : [];
```
in case that `availableDmps === "[]"`, that would effectively give `JSON.parse("[]")`, which is `[]` anyway?